tests/engine/executors/test_executors.py

```diff
@@ -251,6 +169,16 @@ def record_times():
         assert a_start < b_end
         assert b_start < a_end
 
+    def test_context_tags_are_passed_to_submit(self, monkeypatch):
```

Comment:
Bad merge conflict fix. This test is no longer needed, and won't pass as written. Could just delete the whole thing.

Reply:
Dang I knew I was going to miss something - thanks for checking!